### DISCUSSION TITLE
What is [...1] routing syntax?

### DISCUSSION THREAD

**Discussion by stephane-klein** [sveltejs/kit#5452] [Q&A]
Hi,

In `kit-docs` project, [here](https://github.com/svelteness/kit-docs/tree/e4b20bc1d9eb4e2898def10fc583f0bc495f60ef/packages/kit-docs/src/routes/docs) I see this file system route names:

```
[...1]getting-started/
[...2]default-layout/
[...3]markdown/
...
```

I read [SvelteKit routing documentation page](https://kit.svelte.dev/docs/routing#pages) and I didn't find any mention about `[...1]` syntax.  
What is it? It's only to define the order of file and folder?

Best regards,  
Stéphane

Question cross posted in Reddit: https://old.reddit.com/r/sveltejs/comments/vvohs5/what_is_1_routing_syntax_in_sveltekit/

**Comment by stephane-klein** [comment]
I think it is KitDocs specific syntax:

- https://kit-docs.svelteness.dev/docs/getting-started/quickstart#page-ordering
- https://github.com/svelteness/kit-docs/blob/e4b20bc1d9eb4e2898def10fc583f0bc495f60ef/packages/kit-docs/src/node/utils/fs.ts#L44

**Reply by maiertech** [reply]
This is a trick to use rest parameters (which you don't use for anything) to sort files without affecting their corresponding routes.

**Comment by thenbe** [comment]
https://kit.svelte.dev/docs/routing#advanced-routing
